<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mail</class>
 <widget class="QDialog" name="Mail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Send email</string>
   </property>
   <widget class="QLineEdit" name="le_rec">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Recipients</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_obj">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Object</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_desc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_send">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
